sdk/spring/spring-cloud-azure-native-configuration/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 4.0.0-beta.4 (Unreleased)
```

Comment:
wrong version

Reply:
Check the CHANGELOG.md files of other libraries, we prefer to maintain one copy of CHANGELOG.